api/v1alpha1/fenceagentsremediation_types.go

```diff
@@ -63,6 +55,25 @@ type FenceAgentsRemediationSpec struct {
 	//+operator-sdk:csv:customresourcedefinitions:type=spec
 	Agent string `json:"agent"`
 
+	// RetryCount is the number of times the fencing agent will be executed
+	//+kubebuilder:default:=100
```

Comment:
huh, that's a lot IMHO. We are talking about issues here when executing the fence agent command. I think there is low chance that it succeeds after some time once it failed, because probably there is either misconfiguration, or an issue with IPMI etc.. I would go with 5, which results in > 5 minutes already, in combination with the default 1m timeout

Reply:
I need to test this one better. Initially my understanding was that aws agent fails a lot (like a lot), but at the end it was the cluster misconfigured and forgot to review the number of retries.

I agree that 5 is a good number